std_msgs/Header header
float32[] lidar_ranges
float32 lidar_angle_min
float32 lidar_angle_max
float32 lidar_angle_increment
sensor_msgs/Image camera_frame
geometry_msgs/Vector3 imu_vector
float32 ultrasonic_distance
